src/QuarkusProjectListener.ts

```diff
@@ -17,7 +17,16 @@ import { commands, workspace } from 'vscode';
 import { getWorkspaceProjectLabels } from './utils/workspaceUtils';
 import { Disposable } from 'vscode-languageclient';
```

Comment:
Remove this line, and import `Disposable` from `vscode` on line 16:

```
import { commands, workspace, Disposable } from 'vscode';
```

Reply:
Same for https://github.com/angelozerr/vscode-quarkus/blob/7c41108b79144231902c10b5fb502009cb74edc4/src/wizards/debugging/terminateProcess.ts#L18

Reply:
fixed